packages/protocol/contracts/L1/libs/LibProving.sol

```diff
@@ -78,7 +78,9 @@ library LibProving {
         returns (uint8 maxBlocksToVerify)
     {
         // Make sure parentHash is not zero
-        if (tran.parentHash == 0) revert L1_INVALID_TRANSITION();
+        if (tran.parentHash == 0 || tran.blockHash == 0 || tran.signalRoot == 0) {
```

Comment:
The only difference with the old implementation is that in all cases, blockHash and signalRoot cannot be zero.